### PR TITLE
docs(v2): suggest Surge for quick deployment

### PR DESCRIPTION
**Pull request by fakela**
## Motivation
My motivation was to include how Surge can also be used to deploy docusaurus project in a matter of minutes.
 I thought it would be a good idea to have Docusaurus users take advantage of this, so I added it to the docs.
**Steps:**
`npx @docusaurus/init@next init my-website classic`
`surge build/`
**if you have a domain name**
`surge build/ yourdomain.com`

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan
usage described in the section above. pretty easy. Also, an example of a docusaurus app deployed using surge 
(http://panicky-observation.surge.sh/)

### CODE REVIEW

File: docs/getting-started-publishing.md [hunk 1]
```diff
@@ -306,3 +306,43 @@ GitHub enterprise installations should work in the same manner as github.com; yo
 | `GITHUB_HOST` | The hostname for the GitHub enterprise server. |
 
 Alter your `siteConfig.js` to add a property `'githubHost'` which represents the GitHub Enterprise hostname. Alternatively, set an environment variable `GITHUB_HOST` when executing the publish command.
+
+## Deploying with Surge
+
+Surge is a [static web hosting platform](https://surge.sh/help/getting-started-with-surge) , it is used to deploy your Docusaurus project from command line in a minute. Deploying your project to surge is easy and it’s also free (including a custom domain and SSL).
```
Comment (yangshun) [inline docs/getting-started-publishing.md]: Please remove the space before the comma

File: docs/getting-started-publishing.md [hunk 2]
```diff
@@ -306,3 +306,43 @@ GitHub enterprise installations should work in the same manner as github.com; yo
...
+
+Deploy your app in a matter of seconds using surge with the following steps:
+
+1. First, install surge using npm by running the following command
+
+```bash
+npm install --global surge
+```
+
+2. Run a single command inside the root directory of your project
```
Comment (yangshun) [inline docs/getting-started-publishing.md]: Does this require the site to be built first? Aka does the `build` directory have to be populated with the generated files?

File: docs/getting-started-publishing.md [hunk 3]
```diff
@@ -306,3 +306,43 @@ GitHub enterprise installations should work in the same manner as github.com; yo
...
+
+```bash
+surge build/
+```
+
+This generate the version of the site you want to publish in the `build` directory. A site `url` would be given at the end of the upload which can be edited if you want. Done! You will be given a `*.surge.sh subdomain`.
```
Comment (yangshun) [inline docs/getting-started-publishing.md]: > This generate the version of the site you want to publish in the `build` directory.

I don't understand this. Mind elaborating?

File: docs/getting-started-publishing.md (hunk 2 above)
Comment (fakela) [reply docs/getting-started-publishing.md]: Yes

File: docs/getting-started-publishing.md (hunk 3 above)
Comment (fakela) [reply docs/getting-started-publishing.md]: made some edits to the docs

**Pull request closed without merging**